test/penalizer/Utils.ts

```diff
@@ -0,0 +1,175 @@
+import { ether } from '@openzeppelin/test-helpers'
+import { Transaction } from 'ethereumjs-tx'
+import { ethers } from 'ethers'
+import { toChecksumAddress } from 'web3-utils'
+import { RelayRequest } from '../../src/common/EIP712/RelayRequest'
+import TypedRequestData, { getDomainSeparatorHash } from '../../src/common/EIP712/TypedRequestData'
+import { Commitment } from '../../src/enveloping/Commitment'
+import { AccountKeypair } from '../../src/relayclient/AccountManager'
+import { Address } from '../../src/relayclient/types/Aliases'
+import { RelayHubInstance, TestVerifierEverythingAcceptedInstance, TestTokenInstance, IForwarderInstance } from '../../types/truffle-contracts'
+import { getLocalEip712Signature } from '../../Utils'
+
+const gasPrice = '1'
```

Comment:
this is the same as in other tests... although it's never been particularly clear to me why this is the case.

i suppose it must match the `node.conf` value :man_shrugging: